beets/library.py

```diff
@@ -1467,8 +1467,10 @@ def tmpl_aunique(self, keys=None, disam=None):
 
         keys = keys or 'albumartist album'
         disam = disam or 'albumtype year label catalognum albumdisambig'
+        bracket = bracket or '[]'
         keys = keys.split()
         disam = disam.split()
+        bracket = None if bracket is ' ' else list(bracket)
```

Comment:
The `bracket == ' '` is to check for a white space in the third argument for `%aunique` in order to remove any bracketing.